yaksh/forms.py

```diff
@@ -150,7 +150,8 @@ def save(self):
             new_profile.key_expiry_time = timezone.now() + \
                                         timezone.timedelta(minutes=20)
         new_profile.save()
-        return u_name, pwd, new_user.email, new_profile.activation_key
+        user_activation_key = Profile.objects.get(user=new_user).activation_key
```

Comment:
Isn't <code> user_activation_key </code> the same as <code>new_profile.activation_key</code>.